WebApi/Triggers/TreeTriggers.cs

```diff
@@ -1,60 +1,69 @@
-
-            Dictionary<(string,string),ForestGEO.WebApi.Model.Storage.TreeStorage> TreeRecords = mySql.QueryTreeDB("select TempID, QuadratName, Tag, StemTag, Mnemonic as SpCode, DBH, Codes, HOM, ExactDate, x, y, PlotID, CensusID, Errors from tempnewplants");
-
-            var ResponseBuilder = new ArrayList();
-            foreach(var Tree in data){
-                if(TreeRecords.ContainsKey((Tree.Tag, Tree.StemTag)))
-                {
-                    var newTree = new ForestGEO.WebApi.Model.Storage.TreeStorage(Tree);
-                    if(newTree.IsAlive() && TreeRecords[(Tree.Tag, Tree.StemTag)].IsDead())
-                    {
-                        ResponseBuilder.Add(new TreeResponse(Tree, 1, "This tree was dead in a previous census."));
-                    }
-                }
-            }
-            return new OkObjectResult(JsonConvert.SerializeObject(ResponseBuilder));
-        }
-    }
-}
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Microsoft.Azure.Cosmos;
+using ForestGEO.WebApi.Model.Contracts;
+using System.Collections.Generic;
+using ForestGEO.WebApi.Model.Utilities;
+
+namespace ForestGEO.WebApi.Triggers.Tree
+{
+    public static class TreeTriggers
+    {
+        private static readonly string EndpointUri = System.Environment.GetEnvironmentVariable("CosmosEndpointUri", EnvironmentVariableTarget.Process);
+        private static readonly string PrimaryKey = System.Environment.GetEnvironmentVariable("CosmosPrimaryKey", EnvironmentVariableTarget.Process);
+        private static string databaseId = System.Environment.GetEnvironmentVariable("CosmosDatabaseId", EnvironmentVariableTarget.Process);
+        private static string containerId = System.Environment.GetEnvironmentVariable("CosmosContainerId", EnvironmentVariableTarget.Process);
+
+        private static CosmosClient cosmosClient = new CosmosClient(EndpointUri, PrimaryKey);
+        private static readonly Database database = TreeTriggers.cosmosClient.GetDatabase(TreeTriggers.databaseId);
+        private static readonly Container container = TreeTriggers.database.GetContainer(TreeTriggers.containerId);
+
+        [FunctionName("GetCensus")]
+        public static async Task<IActionResult> GetCensus(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = "Census")] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("C# HTTP trigger function processed a get census request.");
+
+            List<Model.Contracts.Tree> trees = await CosmosController.QueryCosmos(
+                TreeUtilities.TreeSQLQuery(
+                    plotId: 1,
+                    censusId: 3), 
+                TreeTriggers.container);
```

Comment:
I think it could be helpful to future readers to include a comment about why these hardcoded values are what they are.